com.github.dnadolny.javatoscala/src/com/github/dnadolny/javatoscala/convertsnippet/PasteJavaAsScalaCommand.scala

```diff
@@ -12,6 +10,7 @@ import org.eclipse.swt.widgets.Display
 import org.eclipse.text.edits.ReplaceEdit
 import org.eclipse.ui.handlers.HandlerUtil
 import org.eclipse.ui.PlatformUI
+import org.scalaide.ui.internal.editor.ScalaSourceFileEditor
```

Comment:
Ah yup it works just referencing `ITextEditor`, so I'll just use that - my rules in `plugin.xml` already take care of making sure we're in a scala editor.